src/platform/FreeBSD.cc

```diff
@@ -31,14 +31,16 @@
 #include "platform/FreeBSD.hh"
 #include "CommandLine.hh"
 #include "UserError.hh"
-
+#include<fcntl.h>
 #include <cassert>
 #include <sstream>
-
-#include <fcntl.h>
+#include<err.h>
 #include <stdlib.h>
 #include <unistd.h>
+#include<libpreopen.h>
+#include<string>
 
+using namespace std;
```

Comment:
I don't think you actually need this `using namespace` directive: as far as I can see, you only use `std::string`, and you refer to that with an explicit namespace.